app/src/main/java/com/github/se/eduverse/ui/camera/PicTakenScreen.kt

```diff
@@ -105,13 +106,14 @@ fun PicTakenScreen(
 
           // Crop and settings icons
           Column(
-              modifier = Modifier.align(Alignment.TopEnd).padding(top = 40.dp, end = 16.dp),
-              verticalArrangement = Arrangement.spacedBy(16.dp)) {
+              modifier = Modifier.align(Alignment.TopEnd),
+              verticalArrangement = Arrangement.spacedBy(30.dp)) {
+                Spacer(modifier = Modifier.height(0.5.dp))
```

Comment:
If feel like those buttons are too close to the edge of the screen, and that they are better with an end padding

Reply:
It were already the case before, the only thing that I have changed is the distance from the top and the icons.